com.unity.render-pipelines.universal/Shaders/Particles/ParticlesSimpleLit.shader

```diff
@@ -116,14 +116,15 @@ Shader "Universal Render Pipeline/Particles/Simple Lit"
             #pragma multi_compile _ _ADDITIONAL_LIGHTS_VERTEX _ADDITIONAL_LIGHTS
             #pragma multi_compile _ _ADDITIONAL_LIGHT_SHADOWS
             #pragma multi_compile _ _SHADOWS_SOFT
+            #pragma multi_compile _SHADER_QUALITY_LOW _SHADER_QUALITY_MEDIUM _SHADER_QUALITY_HIGH
 
             // -------------------------------------
             // Unity defined keywords
             #pragma multi_compile_fog
 
             #pragma vertex ParticlesLitVertex
             #pragma fragment ParticlesLitFragment
-            #define BUMP_SCALE_NOT_SUPPORTED 1
+            #define BUMP_SCALE 0
```

Comment:
if we set this in shader we don't need to expose it in global quality tiers, I belive this should be decied per-shader not per global quality. let's discuss in face to face.